pkg/driver/node/mounter/pod_mounter_test.go

```diff
@@ -522,6 +522,194 @@ func TestPodMounter(t *testing.T) {
 			}
 		})
 
+		t.Run("Mount arg policy: strips --cache-xz flag", func(t *testing.T) {
```

Comment:
All ignored options' test cases could likely be factorized for compacity.

Reply:
Thanks, changed, will be useful later on as well.